.circleci/config.yml

```
@@ -219,19 +224,42 @@ jobs:
                 git checkout --force -B "$CIRCLE_BRANCH" "$CIRCLE_SHA1"
                 git --no-pager log --no-color -n 1 --format='HEAD is now at %h %s'
             fi
+      - run: cat package*.json > cachekey
+      - restore_cache:
+          keys:
+            - v7-root-npm-{{ checksum "cachekey" }}
       - run:
-          name: Lint and test root
-          command: |
-            npm ci
-            npm run lint:scripts
-            npm run lint:yaml
-            npm run test:root
-            rm -rf .git/
+          name: Install root dependencies
+          command: npm ci
+      - save_cache:
+          key: v7-root-npm-{{ checksum "cachekey" }}
+          paths:
+            - ~/.npm
+      - run:
+          name: Remove .git folder
+          command: rm -rf .git/
       - persist_to_workspace:
           root: ~/pix
           paths:
             - .
 
+  root_lint_and_test:
+    executor:
+      name: node-docker
+    working_directory: ~/pix
+    steps:
+      - attach_workspace:
+          at: ~/pix
+      - run:
+          name: Lint root scripts
+          command: npm run lint:scripts
```

(hors scope PR)
C'est pas ouf car on doit ajouter chaque tâche à la main
Je propose de créer la tâche npm  `lint:root`: `npm run lint:scripts && npm run lint:yaml"
```suggestion
      - run:
          name: Lint scripts and yaml
          command: npm run lint:root
```

---

Yes ça a du sens